deploy/cloudfront.tf

```diff
@@ -0,0 +1,62 @@
+locals {
```

Comment:
I think assigning a global variable to a local defeats the point, should probably use one or the other

Reply:
done